csm_web/logistinator/models.py

```diff
@@ -1,3 +1,108 @@
+import datetime
 from django.db import models
+from django.conf import settings
+from django.utils import timezone
 
-# Create your models here.
+
+class Availability(models.Model):
+    """
+    Byte order is little endian, but the days are big endian.
+
+    000 ... 111
+    MON ... SUN
+
+    Within a day, it is also little endian
+
+    8AM 9AM ... 12AM
+    000 111 ... 111
```

Comment:
Each hour corresponds to **four** bits not three, right?

Reply:
This was just arbitrary but I can see why it will be confusing.